script.js

```diff
@@ -0,0 +1,89 @@
+let potionDecline = "You declined sorceress offer of strength potion"
+let villagePeace = "You decided to find peaceful solution with the rogue leader"
+let dragonCompromise = "You decided to find a compromise with a dragon instead of a fight with him"
+let dragonFight = "You decided to fight with the dragon and died but become a legend"
+let mistake = "you did a mistake, try again"
```

Comment:
I'd specify, what is mistake
```suggestion
let mistakeMessage = "you did a mistake, try again"
```